clients/java/src/test/java/io/camunda/zeebe/client/resource/DeployResourceTest.java

```diff
@@ -191,12 +192,14 @@ public void shouldDeployMultipleResources() {
         .newDeployResourceCommand()
         .addResourceFromClasspath(filename1)
         .addResourceFromClasspath(filename2)
+        .tenantId(tenantId)
```

Comment:
🔧 IMO it's better to add additional test cases to verify the multi-tenant deployment functionality instead of adding it to existing cases. The existing cases can work with the default tenant. Setting a specific tenant could be a separate test case. This prevents the test cases from expanding the number of ways they can fail, and when they fail due to multi-tenancy problems this is caught by a specific test case that highlights the problem. It also helps to test the two different cases (with and without explicit tenant)

Reply:
@korthout I won't be able to add an explicit case for the same reason as https://github.com/camunda/zeebe/pull/14089#discussion_r1317028552, i.e. we need to expose the `multiTenancy` configuration flag to be able to test it. Currently, multi-tenancy is disabled internally in the gateway.

I will add a test case and `@Ignore` it until #14041 is implemented.